src/main.js

```diff
@@ -1,6 +1,6 @@
 import { createApp } from 'vue'
 import App from './App.vue'
 
-import './assets/main.css'
+import '../src/assets/styles/main.scss'
```

Comment:
Use `@` path to `assets` to make use of what is defined in `vite.config.js` ->  `resolve.alias`